webapi/src/main/scala/org/knora/webapi/responders/admin/ListsResponderADM.scala

```diff
@@ -417,4 +455,159 @@ class ListsResponderADM extends Responder {
             }
         } yield NodePathGetResponseADM(nodelist = makePath(queryNodeIri, nodeMap, parentMap, Nil))
     }
+
+
+    /**
+      * Creates a list.
+      *
+      * @param createListRequest the new list's information.
+      * @param requestingUser the user that is making the request.
+      * @param apiRequestID   the unique api request ID.
+      * @return a [[ListInfoGetResponseADM]]
+      * @throws ForbiddenException in the case that the user is not allowed to perform the operation.
+      * @throws BadRequestException in the case when the project IRI or label is missing or invalid.
+      */
+    private def listCreateRequestADM(createListRequest: CreateListApiRequestADM, requestingUser: UserADM, apiRequestID: UUID): Future[ListGetResponseADM] = {
```

Comment:
Does this check permissions? (Who has permission to create a list?)

Reply:
Good catch. No, forgot to implement this. I would say that the user needs to be a project admin.